jujugui/static/gui/src/app/components/user-profile/user-profile.js

```diff
@@ -55,6 +56,24 @@ YUI.add('user-profile', function() {
     },
 
     componentWillMount: function() {
+      if (this.props.authenticated) {
```

Comment:
Looks like we want to get the models, but not the charms and bundles at this point.